spec/job/adapters/kubernetes/batch_spec.rb

```diff
@@ -568,6 +569,69 @@ def build_script(opts = {})
 
       @basic_batch.submit(script)
     end
+
+    it "saves pod YAML" do
+      tmp = Dir.mktmpdir
+      pod_yml_path = File.join(tmp, "pod.yml")
```

Comment:
I think we should pass a block here so we ensure it get's cleaned up.

```ruby
Dir.mktmpdir do |tmp|
  pod_yml_path = File.join(tmp, "pod.yml")
  ...
end
```

Reply:
This change is made, wrapped the entire test block for `Dir.mktmpdir`.